### PR TITLE
Messenger profile support

### PR DESCRIPTION
**Pull request by Ltty**

### CODE REVIEW

File: lib/BootBot.js [hunk 1]
```diff
@@ -146,8 +146,8 @@ class BootBot extends EventEmitter {
     .catch(err => console.log(`Error sending message: ${err}`));
   }
 
-  sendThreadRequest(body, method) {
-    return this.sendRequest(body, 'thread_settings', method);
+  sendProfileRequest(body, method) {
+    return this.sendRequest(body, 'messenger_profile', method);
```
Comment (Charca) [inline lib/BootBot.js]: I'd like to keep the `sendThreadRequest` method around for now (in addition to the new `sendProfileRequest`). I'd add a console log in `sendThreadRequest` warning the user that this method has been deprecated and that it'll be removed in a future release. It's unlikely that anyone is using this method directly, but I don't want to introduce any breaking changes if possible.

**Comment by Ltty** [comment]
Hi @Charca,
sorry for my late response. I've been travelling during the past days. I gave you permissions to fork from our repo. I might have time to make your change requests later this week, but if you are faster I'd appreciate your support.

**Comment by sotirelisc** [comment]
This is going to support the new persistent menu, right?

**Comment by Charca** [comment]
@sotirelisc yessir, I still need to add a few tests to this PR but it will be ready to merge soon.

**Comment by Charca** [comment]
I've finished working on this one. I'm going to merge it and release v1.0.12 with it.

**Pull request merged**